src/components/Homepage/Nav/DropdownMenu.js

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from 'react';
+import * as styles from '../../../styles/DropdownMenu.module.css';
+
+const initialCitiesState = [
+  { name: 'Sofia', id: Math.random() },
+  { name: 'Athens', id: Math.random() },
+  { name: 'Istanbul', id: Math.random() }
+];
```

Comment:
you can just hard code something instead of using math.random - you'd need the id to be a constant value